py/server/deephaven/time.py

```diff
@@ -28,6 +28,13 @@
 _NANOS_PER_SECOND = 1000000000
 _NANOS_PER_MICRO = 1000
 
+TimeZoneAlias = Union[TimeZone, str, datetime.tzinfo, datetime.datetime, pandas.Timestamp]
+LocalDateAlias = Union[LocalDate, str, datetime.date, datetime.datetime, numpy.datetime64, pandas.Timestamp]
+LocalTimeAlias = Union[LocalTime, int, str, datetime.time, datetime.datetime, numpy.datetime64, pandas.Timestamp]
+InstantAlias = Union[Instant, int, str, datetime.datetime, numpy.datetime64, pandas.Timestamp]
+ZonedDateTimeAlias = Union[ZonedDateTime, str, datetime.datetime, numpy.datetime64, pandas.Timestamp]
+DurationAlias = Union[Duration, int, str, datetime.timedelta, numpy.timedelta64, pandas.Timedelta]
+PeriodAlias = Union[Period, str, datetime.timedelta, numpy.timedelta64, pandas.Timedelta]
```

Comment:
1) These need pydocs for sphinx.
2) I'm concerned that "Alias" might be confusing to users.  Maybe "Union" or something else.

Reply:
To me, `Alias` is appropriate except for `int`, `str`.  The same is for `Type`, `Kind`.  I don't like `Union` that much either. I don't have a good suggestion and if I am forced to choose, I would pick one of 'Alias', 'Type', and 'Kind'.